src/main/java/tqs/example/impostor/service/AdminService.java

```diff
@@ -0,0 +1,60 @@
+package tqs.example.impostor.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import tqs.example.impostor.repository.*;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@Service
+public class AdminService implements AdminServiceInterface{
+
+    @Autowired
+    private AdminRepository adminRepository;
+
+    @Autowired
+    private OrderRepository orderRepository;
+
+    @Autowired
+    private ACPRepository acpRepository;
+
+    @Override
+    public String loginPage() {
+        return "admin_login";
+    }
```

Comment:
Violates single responsibility principle. The service is not responsible for providing any logic behind the webpages or http responses.